src/App.css

```diff
@@ -1,38 +1,207 @@
-.App {
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: black;
+  color: white;
+}
+
+.header {
+  padding: 7px;
+  background: #ec4e8a;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-actions {
+  display: flex;
+  gap: 10px;
+}
+
+.hero {
+  padding: 30px;
+  text-align: right;
+  height: 40vh;
+  background: rgb(0, 0, 0);
+  background: linear-gradient(90deg, rgba(0, 0, 0, 0.5331632653061225) 8%, rgba(236, 78, 138, 0.4285714285714286) 100%);
+}
+
+.hero h3 {
+  font-size: 40px;
+}
+
+.hero small {
+  font-size: 14px;
+  font-style: italic;
+}
+
+.hero p {
+  font-size: 22px;
+  font-style: italic;
+}
+
+.home_h2 {
   text-align: center;
+  padding: 20px 0;
+  font-size: 17px;
+  font-style: italic;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+a {
+  text-decoration: none;
+  color: white;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+button {
+  background: none;
+  border: none;
+  color: white;
+  font-size: 14px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.hero-cover {
+  position: absolute;
+  z-index: -1;
+  width: 100%;
+  height: 40vh;
+  background: url(./images/angel.jpeg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.main-container {
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  margin-bottom: 20px;
+}
+
+.card {
+  cursor: pointer;
+  height: 20vh;
+  width: 50%;
+  position: relative;
+  background: #ec4e8a;
+}
+
+.card h3 {
+  font-size: 20px;
+}
+
+.card:hover {
+  background: linear-gradient(90deg, rgba(0, 0, 0, 0.5331632653061225) 8%, rgba(236, 78, 138, 0.4285714285714286) 100%);
+}
+
+.card-cover {
+  position: absolute;
+  height: 20vh;
+  background: url(./images/angel.jpeg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.card.ligth-card {
+  background: #cd4275 !important;
+}
+
+.card.ligth-card:hover {
+  background: linear-gradient(90deg, rgba(0, 0, 0, 0.082) 8%, rgba(236, 78, 138, 0.4285714285714286) 100%) !important;
```

Comment:
- [ ] Please do not overuse !important rule in your CSS style definitions, as this is not a good practice when writing code 👍🏿